### ISSUE TITLE
AIT Crashes on multiplayer servers

### ISSUE THREAD

**Issue by Duzos** [M-D-Team/AIT#5]
> Reported by Yoriichi.#1377
Seemingly because it tries to load the screens which are client side only, causing it to crash(?)
Crash report attached
[Crash](https://github.com/M-D-Team/AIT/files/10809155/crash-2023-02-22_23.15.13-fml.txt)

**Comment by Duzos** [comment]
Able to reproduce on my own server

[Report](https://github.com/M-D-Team/AIT/files/10809263/crash-2023-02-23_00.02.43-fml.txt) is the same

**Comment by Duzos** [comment]
` public static final RegistryObject<Block> TENNANT_MONITOR_BLOCK = BLOCKS.register("tennant_monitor",TennantMonitorBlock::new);
`
in AITBlocks is causing the crash

**Comment by DrTheodor** [comment]
Perhaps, the problem is that `net/minecraft/client/gui/screen/Screen` class is client-side only? Hence the error:
```
java.lang.RuntimeException: Attempted to load class net/minecraft/client/gui/screen/Screen for invalid dist DEDICATED_SERVER
```

**Comment by Duzos** [comment]
Yeah but the screen is defined as OnlyIn(CLIENT) so idk how its loading it

**Comment by Duzos** [comment]
![image](https://user-images.githubusercontent.com/73184526/220988271-d4b35cd8-0a79-451b-9ff9-287287e9d416.png)
see
however theres no check to see if setScreen is being ran on the client only so maybe that could be it

**Comment by Duzos** [comment]
Thats weird though bc they're both specified as being client only
Unsure as to why that would crash it or how we can get around that
